### PR TITLE
Bump Citus version to 9.3.3

### PR DESCRIPTION
**Pull request by onurctirtir**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -28,7 +28,7 @@ env:
     #- TARGET_PLATFORM=ubuntu/xenial
     - TARGET_PLATFORM=ubuntu/focal
 before_install:
-  - git clone -b v0.7.17 --depth 1 https://github.com/citusdata/tools.git
+  - git clone -b v0.7.18 --depth 1 https://github.com/citusdata/tools.git
```
Comment (hanefi) [inline .travis.yml:31]: I don't see any changes that are relevant to this branch. Why did you want to bump the tools version?

It's totally ok, I am just curious

File: .travis.yml [hunk 2]
```diff
@@ -28,7 +28,7 @@ env:
     #- TARGET_PLATFORM=ubuntu/xenial
     - TARGET_PLATFORM=ubuntu/focal
```
Comment (hanefi) [inline .travis.yml:29]: It appears that many distros are commented out in here. Please fix this, and do another push

File: .travis.yml (hunk 1 above)
Comment (onurctirtir) [reply .travis.yml:31]: yeah, just I didn't want to lose the track

File: debian/changelog [hunk 3]
```diff
@@ -1,3 +1,9 @@
+citus (9.3.3.citus-1) stable; urgency=low
+
+  * Fixes a memory leak in subtransaction memory handling
+
+ -- Onur Tirtir <Onur.Tirtir@microsoft.com>  Mon, 13 Jul 2020 8:47:4 +0000
```
Comment (onurctirtir) [inline debian/changelog]:
```suggestion
 -- Onur Tirtir <Onur.Tirtir@microsoft.com>  Mon, 13 Jul 2020 8:47:40 +0000
```

**Review by hanefi — APPROVED** [review]
:shipit:

File: debian/changelog (hunk 3 above)
Comment (hanefi) [reply debian/changelog]: nice catch

**Pull request merged**